src/renderer/views/Hashtag/Hashtag.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div>
+    <ft-loader
+      v-if="isLoading"
+      :fullscreen="true"
+    />
+    <div v-else>
+      <ft-card>
+        <h1>{{ hashtag }}</h1>
```

Comment:
Noticed that this title looks a lot larger than other places in FreeTube, turns out those use `h3` instead of `h1`.
```suggestion
        <h3>{{ hashtag }}</h3>
```